src/blocks/food-and-drinks/test/food-and-drinks.cypress.js

```diff
@@ -62,7 +62,9 @@ describe( 'Block: Food and Drinks', function() {
 			cy.get( '[data-type="coblocks/food-item"]' ).first().find( '.block-editor-media-placeholder' ).should( 'not.exist' );
 		} );
 
-		helpers.selectBlock( 'food & drink' );
```

Comment:
Why are we moving away from using helpers here?

Reply:
Good catch. I actually meant to update the helper function with the new method after verifying this new code was working.

Reply:
And apparently I didn't realize it is the way the helper function works... I'm not really sure why I did this now. Either way, I'm updating it to use the helper.